config/initializers/breakers.rb

```diff
@@ -84,6 +85,7 @@
     SearchTypeahead::Configuration.instance.breakers_service,
     SearchClickTracking::Configuration.instance.breakers_service,
     VAOS::Configuration.instance.breakers_service,
+    Vye::DGIB::Configuration.instance.breakers_service,
```

Comment:
Vye::DGIB::Service.instance.breakers_service,

Reply:
Are you saying to add this as a new entry or replace Vye::DGIB::Configuration.instance.breakers_service?

Reply:
Replace Vye::DGIB::Configuration.instance.breakers_service with Vye::DGIB::Service.instance.breakers_service

Reply:
change made